deepmd/pt/infer/deep_eval.py

```
@@ -598,3 +600,229 @@ def eval_typeebd(self) -> np.ndarray:
+    logits_out = []
+    err_msg = (
+        f"All inputs should be the same format, "
+        f"but found {type(coords)}, {type(cells)}, {type(atom_types)} instead! "
+    )
+    return_tensor = True
+    if isinstance(coords, torch.Tensor):
+        if cells is not None:
+            assert isinstance(cells, torch.Tensor), err_msg
+        if spins is not None:
+            assert isinstance(spins, torch.Tensor), err_msg
+        assert isinstance(atom_types, torch.Tensor) or isinstance(atom_types, list)
```

Ruff

Multiple `isinstance` calls for `atom_types`, merge into a single call

Merge `isinstance` calls for `atom_types`

(SIM101)